tidb-in-kubernetes/reference/components/tidb-scheduler.md

```diff
@@ -0,0 +1,60 @@
+---
+title: TiDB Scheduler
+category: components
+Summary: Learn how TiDB Scheduler works
+---
+
+# TiDB Scheduler
+
+This document introduces how TiDB Scheduler works.
```

Comment:
```suggestion
TiDB Scheduler is a TiDB implementation of [Kubernetes scheduler extender](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/scheduler_extender.md). TiDB Scheduler is used to add new scheduling rules to Kubernetes. This document introduces these new scheduling rules and how TiDB Scheduler works.
```
Users might want to know first what is TiDB Scheduler and what TiDB Scheduler does, so I add two sentences to improve this introduction paragraph. 

Please review my suggestions. @weekface